providers/linkedin.go

```diff
@@ -88,6 +88,7 @@ func (p *LinkedInProvider) GetEmailAddress(ctx context.Context, s *sessions.Sess
 	requestURL := p.ProfileURL.String() + "?q=members&projection=(elements*(handle~))"
 	json, err := requests.New(requestURL).
 		WithContext(ctx).
+		WithClient(p.Client).
```

Comment:
Are there any `requests.New(...)` statements left that don't require a client because if it is a mandatory field anyway. We should extend the "constructor".